### PR TITLE
Add package release instruction

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: 020-package-release-criteria.md [hunk 1]
```diff
@@ -10,3 +12,31 @@
...
+2. Pull last changes of master branch from GIT:
+
+```shell
+./yii-dev git/checkout master package-name
+./yii-dev git/pull package-name
+```
+
+3. Check the package for compliance with the criteria above.
+
+4. Run make release command:
```
[the remark this answers is not in the document]
Comment (samdark) [reply 020-package-release-criteria.md:32]: It is release/make command.

File: 020-package-release-criteria.md [hunk 2]
```diff
@@ -10,3 +12,31 @@
...
+[Signing commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)).
+
+2. Pull last changes of master branch from GIT:
+
+```shell
+./yii-dev git/checkout master package-name
+./yii-dev git/pull package-name
+```
+
+3. Check the package for compliance with the criteria above.
```
[the remark this answers is not in the document]
Comment (samdark) [reply 020-package-release-criteria.md:30]: It is alright as is.

File: 020-package-release-criteria.md [hunk 3]
```diff
@@ -10,3 +12,31 @@
 - Psalm analysis pass on at least level 2.
 - phpdoc is alright.
 - Public API is alright.
+
+## Release Instruction
+
+Release package via [Yii Development Tool](005-development-tool.md). 
+
+1. Check that sign commits locally (see
```
[the remark this answers is not in the document]
Comment (samdark) [reply 020-package-release-criteria.md:20]: No, it is necessary.

File: 020-package-release-criteria.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samdark) [reply 020-package-release-criteria.md:32]:
```suggestion
4. Run `release/make` command:
```

File: 020-package-release-criteria.md [hunk 4]
```diff
@@ -10,3 +12,31 @@
 - Psalm analysis pass on at least level 2.
 - phpdoc is alright.
 - Public API is alright.
+
+## Release Instruction
+
+Release a package via [Yii Development Tool](005-development-tool.md). 
+
+1. Check that sign commits locally (see
```
Comment (samdark) [inline 020-package-release-criteria.md:20]:
```suggestion
1. Check that you can sign commits locally (see 
```

**Review by samdark — APPROVED** [review]

**Pull request merged**